test/ecto/query/subquery_test.exs

```diff
@@ -310,4 +375,33 @@ defmodule Ecto.Query.SubqueryTest do
       normalize(from(p in subquery(subquery), select: [:title]))
     end
   end
+
+  defp macro_arg_at(m, i) do
+    m |> elem(2) |> Enum.at(i)
+  end
```

Comment:
It is best to pattern match instead of elem(2) |> Enum.at. Pattern matching always asserts more about the shape. :)